client/me/purchases/manage-purchase/index.jsx

```diff
@@ -500,6 +541,203 @@ class ManagePurchase extends Component {
 		return null;
 	}
 
+	renderRemoveSubscriptionWarningDialog( site, purchase ) {
+		if ( this.state.showRemoveSubscriptionWarningDialog ) {
+			const { hasCustomPrimaryDomain, primaryDomain } = this.props;
+			const primaryDomainName = hasCustomDomain && primaryDomain ? primaryDomain.name : '';
```

Comment:
`hasCustomDomain` is a funtion imported above as:
```
import { hasCustomDomain } from 'calypso/lib/site/utils';
```
so this condition is actually wrong :)